support/dictionaries/dict-calcite-design-system.txt

```diff
@@ -36,6 +36,7 @@ Steffen
 signifiers
 stringifying
 uppercased
+xsmall
```

Comment:
Would this being under a `## responsive breakpoints` or similar be helpful?

Reply:
Maybe if we want a `## Sizing` category? they seem pretty general though.